pycode/memilio-simulation/memilio/simulation_test/test_date.py

```diff
@@ -0,0 +1,85 @@
+#############################################################################
+# Copyright (C) 2020-2023 German Aerospace Center (DLR-SC)
+#
+# Authors: Maximilian Betz
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+
+import unittest
+
+from memilio.simulation import Date
+
+
+class TestCustomIndexArray(unittest.TestCase):
+    def test_init(self):
```

Comment:
Again, we should test the behaviour with invalid dates here :)
Also, tests for dates in leap years would be beautiful.

Reply:
Firstly, i think those test are redundant, because that is already done in cpp. Secondly asserts arent triggered when using pycode simulation module.